src/index.ts

```diff
@@ -1,7 +1,6 @@
-import * as posthtml from "posthtml";
```

Comment:
one of the issue came from this one, not in the dependencies but also not used in this file ^^